RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronID_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+lowPtGsfElectronID = cms.EDProducer(
```

Comment:
This is a duplicate file. Since you are not modifying any of the default parameter here, it can just be removed (simpler solution, if you don't plan to modify those parameters via config switches).

If you think that some of those parameters can be eventually modified via configuration, then you can either add the "default" string (or whatever else you like) in the config automatically created by fillDescriptions() and import and clone that default sequence; or alternativeli rename this file as lowPtGsfElectronID_cff.py, make it  import the  lowPtGsfElectronID defined there, and replace everywhere else the inclusion of lowPtGsfElectronID_cfi.py with the inclusion of lowPtGsfElectronID_cff.py.